index.html

```diff
@@ -17,18 +17,20 @@
   <link rel="stylesheet" href="./styles/mobile..css">
 </head>
 <body>
-  <nav class="mobile-nav">
+  <header class="mobile-header">
     <div class="logo">
       <h2 class="name">Rassel</h2>
     </div>
-    <div class="hum">
-      <img src="./images/hum.png" alt="">
-    </div>
-  </nav>
+    <nav class="mobile-nav">
+      <div class="hum">
+        <img src="./images/hum.png" alt="">
+      </div>
+    </nav>
+  </header>
 
   <section class="container">
     <div class="primary-text">
-      <h2 class="greetings">Hey there. I'm Rassel</h2>
+      <h1 class="greetings">Hey there. I'm Rassel</h1>
```

Comment:
- [ ] [OPTIONAL] Kindly reduce the space a bit to make it more pixel-perfect, as you shall be doing in your soon coming life-changing future job 😀.